pseudocode.py

```diff
@@ -0,0 +1,126 @@
+'''
+Aerolyzer - Pseudocode
+Feburary 10, 2017
+
+
+Image Restrictions:
+
+Throughout the upload process, images will be required to meet various
+criteria beforee being accepted for aerosol content analysis. To accomplish this
+requirement, a mulititude of functions will be created, each aimed at verifiying
+that a certain restriction has been met. To ensure the simplicity of Aerolyzer's
+initial development, as well as future adaptations, information regarding image criteria
+will be stored in a single file to be referenced by the various restriction functions
+outlined below.
+'''
+
+'''
+Purpose:        The purpose of this file is to keep all image criteria specifications located in
+                one place, for simplicity's sake.
+Inputs:         None
+Outputs:        None
+Returns:        None
+Assumptions:    These values will be populated by a developer.
+'''
+acceptedFileTypes = ["JPEG", "JPG", "PNG"]
+acceptedMobileDevices = ["i5", "i5s", "i6", "i6s", "i7"]
+imgMaxSize = 200;
+imgWidthMin = xxx;
+imgHeightMin = xxx;
+
+'''
+Purpose:        The purpose of this function is to determine whether or not the device the
+                image was taken on is an accepted mobile device.
+Inputs:         dict exifData, list acceptedMobileDevices
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def accept_device(exifData, acceptedMobileDevices):
+    for device in range(len(acceptedMobileDevices)):
+        if (exifData[DEVICE] == acceptedMobileDevices[device]):
+            return true
+    return false
+
+'''
+Purpose:        The purpose of this function is to determine whether or not the image was
+                altered from its original form. I.e. do the modification and creation dates coincide.
+Inputs:         dict exifData
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def not_edited(exifData):
```

Comment:
In keeping with the general convention for boolean functions, this function should be called:
is_edited